Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -508,3 +512,4 @@ private extension NSError {
         return domain == NSCocoaErrorDomain && (code == NSFileReadNoSuchFileError || code == NSFileNoSuchFileError)
     }
 }
+// swiftlint:enable file_length
```

Comment:
this isn't necessary because SwiftLint works on a per-file basis  😉 

Also, on our current project we are also only disabling it at the beginning of the file